src/scripts/copy_all.sh

```diff
@@ -0,0 +1,43 @@
+#Coreutils
+cd /home/lind/lind_project/tests/applications/coreutils/src;
+find . -perm /a+x -type f -exec lindfs cp /home/lind/lind_project/tests/applications/coreutils/src/{} /bin/{} \;
+
+#Bash
+lindfs cp /home/lind/lind_project/tests/applications/bash/bash /bin/bash
+
+#Python
+lindfs cp /home/lind/lind_project/tests/applications/python/python /bin/python;
+lindfs cp /home/lind/lind_project/tests/applications/python/libpython2.7.so.1.0 /lib/glibc/libpython2.7.so.1.0
+lindfs cp /home/lind/lind_project/tests/applications/python/Lib /usr/local/lib/python2.7
+lindfs cp /usr/lib/python2.7/config/ /usr/lib/python2.7/config/
+lindfs cp /usr/include/python2.7/ /usr/include/python2.7/
+lindfs cp /home/lind/lind_project/src/scripts/includes/passwd /etc/passwd
+
+#Python Libs
+echo Copying files...
+libsarray=('_collections' 'operator' 'itertools' '_struct' 'math' 'binascii' 'time' 'cStringIO' '_random' 'array' '_socket' '_functools' 'cPickle' '_struct' 'select' 'array' 'unicodedata')
+length=${#libsarray[@]}
+for (( i = 0; i < length; i++ ));
```

Comment:
you should just do for var in libsarray and use $var within the loop

Reply:
Done.